jms/src/test/scala/docs/scaladsl/JmsBufferedAckConnectorsSpec.scala

```diff
@@ -380,5 +382,47 @@ class JmsBufferedAckConnectorsSpec extends JmsSharedServerSpec {
       }
       resultList.toSet should contain theSameElementsAs numsIn.map(_.toString)
     }
+
+    "flush acknowledgments to broker after flush.timeout triggers" in withConnectionFactory() { connectionFactory =>
```

Comment:
Give this test a more meaningful name now that we've settled on terms for max-ack-interval.